### ISSUE TITLE
error when calling model() on greta@dev

### ISSUE THREAD

**Issue by jdyen** [greta-dev/greta#222]
Hi Nick, I'm getting the following error (on the latest dev branch version) when running simple models:
```
int <- normal(0, 5)
coef <- normal(0, 3)
sd <- lognormal(0, 3)
mean <- int + coef * iris$Petal.Length
distribution(iris$Sepal.Length) <- normal(mean, sd)
m <- model(int, coef, sd)

Error in py_get_attr_impl(x, name, silent) : 
  AttributeError: 'module' object has no attribute 'distributions'
```

I get a different error with more complex models but haven't got a MRE for this one:
```
 Error in py_call_impl(callable, dots$args, dots$keywords) : 
  ValueError: Error converting shape to a TensorShape: invalid literal for int() with base 10: 'dim2'. 
```

Output from `sessionInfo()`:
```
R version 3.4.0 (2017-04-21)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS  10.13.5

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] greta_0.2.5

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.18      rstudioapi_0.7    whisker_0.3-2    
 [4] magrittr_1.5      hms_0.4.2         progress_1.2.0   
 [7] lattice_0.20-35   R6_2.2.2          rlang_0.2.2      
[10] globals_0.12.2    tools_3.4.0       parallel_3.4.0   
[13] grid_3.4.0        coda_0.19-1       tfruns_1.4       
[16] yaml_2.2.0        digest_0.6.16     assertthat_0.2.0 
[19] crayon_1.3.4      tensorflow_1.9    Matrix_1.2-14    
[22] base64enc_0.1-3   codetools_0.2-15  compiler_3.4.0   
[25] prettyunits_1.0.2 reticulate_1.10   jsonlite_1.5     
[28] future_1.9.0      listenv_0.7.0     pkgconfig_2.0.2  
```

**Comment by goldingn** [comment]
Hmmm, sounds like a tensorflow or TFP version issue. Do you know which versions of those you have, or when you installed them?

**Comment by jdyen** [comment]
tf 1.8. Not sure about TFP.

**Comment by jdyen** [comment]
Fixed with an update to tf 1.10.1 and TFP 0.3.0
